src/app/src/api/boundaries.js

```diff
@@ -21,7 +21,7 @@ const boundaryApi = api.injectEndpoints({
                 url: `/boundaries/${id}/`,
                 method: 'GET',
             }),
-            providesTags: getSingleItemProvider(TAGS.BOUNDARY),
+            // providesTags: getSingleItemProvider(TAGS.BOUNDARY),
```

Comment:
@mstone121 I would be grateful if you have pointers for debugging the failure I saw:

`Uncaught (in promise) TypeError: description(...).map is not a function`

Reply:
This is from my implementation. I forgot that when `providesTags` or `invalidatesTags` is passed a function, it expects that function to return an array of tags rather than a single tags. This diff should fix the issue:

```diff
diff --git a/src/app/src/api/boundaries.js b/src/app/src/api/boundaries.js
index 0199429..c362afd 100644
--- a/src/app/src/api/boundaries.js
+++ b/src/app/src/api/boundaries.js
@@ -21,7 +21,7 @@ const boundaryApi = api.injectEndpoints({
                 url: `/boundaries/${id}/`,
                 method: 'GET',
             }),
-            // providesTags: getSingleItemProvider(TAGS.BOUNDARY),
+            providesTags: getSingleItemProvider(TAGS.BOUNDARY),
         }),
 
         startNewBoundary: build.mutation({
diff --git a/src/app/src/api/tags.js b/src/app/src/api/tags.js
index 9903447..5f3146c 100644
--- a/src/app/src/api/tags.js
+++ b/src/app/src/api/tags.js
@@ -12,15 +12,15 @@ export function getListTagProvider(tag) {
 }
 
 export function getSingleItemProvider(tag) {
-    return (result, error, id) => ({ tag, id });
+    return (result, error, id) => [{ tag, id }];
 }
 
 export function getNewItemTagInvalidator(tag) {
-    return () => ({ tag, id: TAGS.LIST_ID });
+    return () => [{ tag, id: TAGS.LIST_ID }];
 }
 
 export function getUpdateItemTagInvalidator(tag) {
-    return (result, error, id) => ({ tag, id });
+    return (result, error, id) => [{ tag, id }];
 }
 
 export default TAGS;